source/getting-started/concepts-terminology.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: "Concepts and terminology"
+description: "User explanation of Home Assistant basics"
+---
+Now you're in Home Assistant, let's go over important parts of Home Assistant and what they do.
+
+### Dashboards
+Customizable pages to display information connected to and available in Home Assistant. Dashboards are the first thing you see after the [onboarding process](/getting-started/onboarding). More details can be found [here](/dashboards).
+
+![Dashboard](/images/getting-started/lovelace.png)
+### Integrations
+Software that allows Home Assistant to connect to other software and platforms. For example, Philips Hue integration allow Home Assistant to talk to the Hue Bridge and read data such as the state of your lights through to controlling the lights themselves.
+
+![Integrations](/images/getting-started/integrations.png)
+
+A full list of compatible integrations can be found [here](/integrations).
+
+Once an integration has been added, the hardware and/or data are represented in Home Assistant as [devices and entities](#devices--entities).
+
+### Devices & Entities
+Devices are logical grouping for one or more entities. A device may represent a physical device and the device may have one or more sensors which appear as entities associated to the device. For example, a motion sensor is represented as a device and the motion may provide motion detection, temperature and light levels as entities. Entities have states such as `on` when motion is detected and `off` where is no motion.
+
+Devices and entities can be used in dashboards to [automations](#automations).
+
+![Home Assistant Device](/images/getting-started/home-assistant-device.png)
+
+### Automations
+A set of repeatable actions that can be set up to run automatically. More details can be found [here](/automation).
+
+![Automations](/images/getting-started/automation-editor.png)
+
+### Scripts
+Similar to automations, repeatable actions that can be run. The difference between scripts and automations is scripts do not have triggers like automations so they cannot automatically run unless it's used in an automation. Scripts are particularly useful if you perform the same actions in different automations or triggering them from a dashboard. More details can be found [here](/integrations/script/).
+
+### Scenes
+Scenes are saved states of entities which can be recalled using scripts or automations. For example, a light can be set to a particular brightness, color, etc. More details can be found [here](/integrations/scene/).
```

Comment:
```suggestion
Scenes are saved states of entities that can be recalled using scripts or automations. For example, a light can be set to a particular brightness, color, etc. More details can be found [here](/integrations/scene/).
```

Reply:
A good bit of context to add here might be: How are those different from automations or scripts?

Reply:
@frenck I have added the sentence:
_Scenes cannot have conditions are actions like scripts and automations._

is that what you mean?

Reply:
Right, yes/no.

So, that makes it sound like scenes are less useful (a more negatively loaded approach, as they can do less than automation and scripts?).

I think the difference between scenes and automation in general, is very unclear in the community. Scenes are often less popular (which is IMHO, not always justified).

Looking at the gist/general question one could have: "Why would I turn on a light using a scene? An automation can do that too?"; which is, of course absolutely true.

(Sorry, thinking out loud here, as I also struggle with a short way to explain it clearly towards an user).

> Scenes are saved states of certain entities that can be recalled using scripts or automation.

Maybe that part is just too technical in this case? "states that can be recalled"?

Or an example could maybe help as well. Like: Set the cosy mood in your living room, by setting all your lights in the mood you are in (brightness, color), setting the thermostat to a nice warm temperature to match the warm lights. By creating a scene for those devices, you now have a cosy scene that you can activate with a single button push that will restore all your device into that cosy state any time in the future... or something better rolling